apicast/src/provider.lua

```diff
@@ -385,8 +385,15 @@ function _M.access(service)
     ngx.var.cached_key = concat({service.id, params.app_id, params.app_key}, ':')
 
   elseif backend_version == 'oauth' then
-    ngx.var.access_token = parameters.access_token
-    params.access_token = parameters.access_token
+    if credentials.location == 'headers' then
+      local token_type, access_token = unpack(split(parameters.authorization or "", " "))
+      if access_token and string.lower(token_type) == "bearer" then
```

Comment:
In the RFCs I've only seen "Bearer". Applying `string.lower` means that we will accept strings such as "bEaRer". Is this what we want? I'm not familiar with the oauth flow, so I'm not sure whether this can be a problem.